### ISSUE TITLE
Proposal: Grafana 7.0 Logo

### ISSUE THREAD

**Issue by x70b1** [algenty/grafana-flowcharting#157]
Grafana 7.0 will be released soon. A new matching logo would be cool.
Unfortunately I dont know which file has to be changed. That is why I have not opened a Pull Request.

I made this with draw.io. 

```
<?xml version="1.0"?>
<svg width="64" height="64" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">

 <g class="layer">
  <title>Layer 1</title>
  <g id="svg_1">
   <path d="m28,32l2,0" fill="none" id="svg_2" stroke="#84aff1" stroke-miterlimit="10" stroke-width="2"/>
   <rect fill="none" height="12" id="svg_3" rx="1.8" ry="1.8" stroke="#f9b11f" stroke-width="3" width="24" x="2" y="26"/>
   <rect fill="none" height="12" id="svg_4" rx="1.8" ry="1.8" stroke="#f3cb0d" stroke-width="3" width="24" x="38" y="14"/>
   <rect fill="none" height="12" id="svg_5" rx="1.8" ry="1.8" stroke="#ff9a2f" stroke-width="3" width="24" x="38" y="38"/>
   <path d="m32,6.5l0,50.15" fill="none" id="svg_6" stroke="#3865ab" stroke-miterlimit="10" stroke-width="3"/>
   <ellipse cx="32" cy="4" fill="#3865ab" id="svg_7" rx="2.5" ry="2.5" stroke="#3865ab" stroke-width="3"/>
   <path d="m32,60.65l-2,-4l4,0l-2,4z" fill="#3865ab" id="svg_8" stroke="#3865ab" stroke-miterlimit="10" stroke-width="3"/>
   <path d="m34,20l2,0" fill="none" id="svg_9" stroke="#84aff1" stroke-miterlimit="10" stroke-width="2"/>
   <path d="m34,44l2,0" fill="none" id="svg_10" stroke="#84aff1" stroke-miterlimit="10" stroke-width="2"/>
  </g>
 </g>
</svg>
```

![screenshot](https://user-images.githubusercontent.com/10016267/81337507-636b3880-90ab-11ea-948d-76f0db3e2c07.png)

**Comment by algenty** [comment]
Nice ! Good idea

**Comment by algenty** [comment]
The files ares in src/img

**Comment by x70b1** [comment]
What is the use case for the files in `dist/img`?

**Comment by algenty** [comment]
src file moved to build after build

**Comment by algenty** [comment]
I posted it on twitter to know advice of users

**Comment by x70b1** [comment]
Closed in #158. Thanks!

**Comment by algenty** [comment]
Welcome to the project
